main.go

```diff
@@ -87,13 +87,14 @@ func runServer() {
 	)
 	if err != nil {
 		log.Error().Err(err).Msg("Logger could not be initialized")
-		os.Exit(1)
+		return
 	}
+	defer logger.CloseZerolog()
 
 	// Storage
 	if _, err = os.Stat(storageConfig.RulesPath); err != nil {
 		log.Error().Err(err).Msg("Storage data path not found")
-		os.Exit(1)
+		return
```

Comment:
what can this service do without data?

Reply:
Nothing. The return finishes the program. Pushed a new commit exiting with code 1 in case there is some error running the server. PTAL.